### PR TITLE
Dataset additions nov 2020

### PR DESCRIPTION
**Pull request by danieltwigt**
Changes related to addition of CRUCIAL and MSFD datasets.

See https://issuetracker.deltares.nl/projects/GDV/issues/GDV-433.

Front-end changes:

- New icons for datasets and themes
- Added text to user agreement related to new datasets
- Two small cosmetic changes to dataset select: naming for layer selection changed, and unit with min and max value of legend removed, since the unit is also shown on the righthand side of each legend.

### CODE REVIEW

**Comment by evetion** [comment]
Depends on https://github.com/openearth/dgds-backend/pull/81

File: components/data-set-controls.vue [hunk 1]
```diff
@@ -43,7 +43,7 @@
             id="layer-options-dropdown"
             v-model="selectedLayer"
             :options="items(dataset.layerOptions)"
-            :label="`Select ${dataset.name}`"
```
Comment (evetion) [inline components/data-set-controls.vue:46]: Why these changes? (if you removed the interpolation, ` aren't required and you could use ", but that's a nitpick)

File: components/layer-legend.vue [hunk 2]
```diff
@@ -18,7 +18,7 @@
     </svg>
     <div class="layer-legend__range unselectable">
       <div v-if="!editingRange" class="layer-legend__range-min">
-        <ui-button kind="quiet" @click="editRange">{{ minValue }}{{ unit }}</ui-button>
```
Comment (evetion) [inline components/layer-legend.vue:21]: Also here, why remove the unit?

**Review by evetion — APPROVED** [review]

File: components/layer-legend.vue (hunk 2 above)
Comment (danieltwigt) [reply components/layer-legend.vue:21]: The unit was show twice: 
- Firstly, below the color bar: {{ minValue }}{{ unit }} and {{ maxValue }}{{ unit }}
- Secondly, also on the right hand side of the color bar.
See for example:
image

This is not required. So either the units shown below the color bar, or the unit shown on the right hand side, could be removed. Opted to remove the unit below the color bar because of this.

Although this change is not essential, this was commented on by on the parties providing the new datasets.

File: components/data-set-controls.vue (hunk 1 above)
Comment (danieltwigt) [reply components/data-set-controls.vue:46]: When using Select ${dataset.name}, this leads to, for example:

image

So underneath the dataset name "Maximum water level", it says "Select maximum water level". Which is not very useful. A text like "Select return period" would be more applicable here.

For the Tidal indicators, it says "Select Tidal indicators", which would be OK.

For the new CRUCIAL data, something like "Select analysis period" would be more applicable.

To change this into something more general, I have now changes this to "Select layer", which is less specific, but applicable to all of the above examples.

**Comment by evetion** [comment]
@CindyvdVries Just a ping, so you're aware of the small changes in here.
